slsim/Sources/point_sources.py

```diff
@@ -24,6 +27,8 @@ def __init__(
         :param sky_area: Sky area over which galaxies are sampled. Must be in units of
             solid angle.
         :type sky_area: `~astropy.units.Quantity`
+        :param kwargs_cut: cuts in parameters: band, band_mag, z_min, z_max
```

Comment:
here you can specify that these are arguments that go into the deflector_cut() definition